virtualizarr/xarray.py

```diff
@@ -142,6 +162,30 @@ def open_virtual_dataset(
                 if name in loadable_variables
             }
 
+            for name in cftime_variables:
+                import cftime
+
+                var = ds[name]
+                calendar = var.attrs.get(
+                    "calendar", var.encoding.get("calendar", "standard")
+                )
+                units = var.attrs.get("units", var.encoding["units"])
+
+                # undoing CF recoding in original input
+                values = []
+                for c in var.values:
```

Comment:
This is a for loop iterating over each value in an array and decoding/encoding each one. Is this really necessary?

Reply:
🤷 I don't think this is a massive issue for now, given that most cftime variables are going to be 1D and therefore likely small.